boards/riscv/rv32m1_vega/doc/index.rst

```diff
@@ -150,7 +150,7 @@ with this board:
    BLE Software Link Layer limitations:
 
    - no 512/256 Kbps PHY
-   - **no LL Privacy**
+   - **no power-save**
```

Comment:
DSM is already merged.
Remove "no power-save" line

Reply:
This line 
"Limitations marked bold above are planned to be addressed on this platform."
is not needed since there are no limitations that we want to address left

Reply:
Thank you for the catch. I made the proper changes.